package.json

```diff
@@ -7,6 +7,8 @@
     "@percy/cli": "^1.0.0-beta.73",
     "@percy/playwright": "^1.0.1",
     "@playwright/test": "^1.18.0",
+    "@types/eventemitter3": "^1.0.0",
+    "@types/lodash": "^4.14.178",
```

Comment:
This doesn't update the packages we're currently using, this only adds types (hence the `@types/`) the package's we're using.